applications/vessel-history/src/features/settings/Settings.tsx

```diff
@@ -45,9 +44,9 @@ const Settings: React.FC = (): React.ReactElement => {
     if (selectedOption) {
       setSelectedOption('')
     } else {
-      dispatch(redirect({ type: HOME }))
+      history().goBack()
```

Comment:
when you are in a section of the settings this not return to that section? @rdgfuentes